pkg/metadata/host/host_tags.go

```diff
@@ -145,7 +145,7 @@ func GetHostTags(cached bool) *Tags {
 	}
 
 	t := &Tags{
-		System:              hostTags,
+		System:              util.SortUniqInPlace(hostTags),
```

Comment:
What do you think about moving the call `util.SortUniqInPlace` inside `GetConfiguredTags`?

Reply:
This way de-duplication would only apply to configured tags before splitting, while duplicates can still occur due to tag splitting via `tag_value_split_separator`, and by having redundant or overlapping tags from k8s/gke/ec2.